eng/pipelines/corefx-base.yml

```
@@ -195,7 +199,7 @@ jobs:
                 helixQueues: $(_helixQueues)
                 msbuildScript: $(_msbuildCommand)
                 framework: $(_framework)
-                outerloop: $(_outerloop)
+                testfilter: $(_testfilter)
```

This should use the parameter value, right? 
```suggestion
          - _testFilterArg: -testfilter ${{ parameters.testFilter }}
```